stdlib/REPL/src/latex_symbols.jl

```diff
@@ -524,6 +524,8 @@ const latex_symbols = Dict(
     "\\setminus" => "∖",
     "\\ast" => "∗",
     "\\circ" => "∘",
+    "\\fatsemi" => "⨟",
+    blackboard*"semi" => "⨟",
```

Comment:
My vote would be to just have the second of these.  `\fatsemi` is less consistent with our other abbreviations, and is hardly standard (it is just the name in one LaTeX add-on package as far as I can tell?)